DogeNetWebClient/src/app/actions/conversation.actions.ts

```diff
@@ -0,0 +1,30 @@
+import { Action } from '@ngrx/store';
+import { ConversationDetailsModel } from 'src/app/core/api/models';
+
+export const SET_CONVERSATIONS = '[Conversations] Set';
+export const ADD_CONVERSATION = '[Conversations] Add';
+export const DELETE_CONVERSATION = '[Conversations] Delete';
+export const CLEAR_CONVERSATIONS = '[Conversations] Clear';
+
+export class SetConversations implements Action {
```

Comment:
This is an old approach to creating actions, please use new one (https://ngrx.io/guide/store/actions)

Reply:
done